### PR TITLE
#74 add delete method into Database managers

### PR DESCRIPTION
**Pull request by kortov**
#74

### CODE REVIEW

File: src/main/java/juja/sqlcmd/InMemoryDatabaseManager.java [hunk 1]
```diff
@@ -83,6 +84,24 @@ boolean insert(DataSet row) {
...
+                return true;
+            }
+            return false;
+        }
+
+        private int findRowById(int id) {
+            for (int index = 0; index < rows.size(); index++) {
+                DataSet currentRow = rows.get(index);
+                String currentRowId = currentRow.values()[0];
+                if (currentRowId.equals(String.valueOf(id))) return index;
```
Comment (Shtramak) [inline src/main/java/juja/sqlcmd/InMemoryDatabaseManager.java]: It's better to use braces here  to reduce the possibility of confusion when reading or modifying the code. And also that is the part of google code style convesion - http://bit.ly/2FUBU7N

File: src/main/java/juja/sqlcmd/InMemoryDatabaseManager.java [hunk 2]
```diff
@@ -83,6 +84,24 @@ boolean insert(DataSet row) {
...
+            }
+            return -1;
+        }
+
         private boolean isRowContainsNull(DataSet row) {
```
Comment (Shtramak) [inline src/main/java/juja/sqlcmd/InMemoryDatabaseManager.java]: Just think about such signature hasRowNullElement(DataSet row)

**Review by Shtramak — APPROVED** [review]
Well done!

File: src/main/java/juja/sqlcmd/InMemoryDatabaseManager.java (hunk 1 above)
Comment (AVM82) [reply src/main/java/juja/sqlcmd/InMemoryDatabaseManager.java]: Agree with Shtramak

Comment (AVM82) [reply src/main/java/juja/sqlcmd/InMemoryDatabaseManager.java]: Agree with Shtramak

**Review by AVM82 — APPROVED** [review]

**Pull request merged**